packages/mdc-ripple/util.ts

```diff
@@ -154,16 +154,17 @@ function getNormalizedEventCoords(ev, pageOffset, clientRect) {
   let normalizedY;
   // Determine touch point relative to the ripple container.
   if (ev.type === 'touchstart') {
-    ev = /** @type {!TouchEvent} */ (ev);
-    normalizedX = ev.changedTouches[0].pageX - documentX;
-    normalizedY = ev.changedTouches[0].pageY - documentY;
+    const e = ev as TouchEvent;
```

Comment:
Can we give this variable a more descriptive name like `touchEvent` rather than an even more abbreviated name? Single-letter variables are notoriously harder to search usages of.